File: src/main/java/com/github/frcsty/frozenactions/actions/broadcast/ActionbarBroadcastAction.java
package com.github.frcsty.frozenactions.actions.broadcast;

import com.github.frcsty.frozenactions.actions.Action;
import com.github.frcsty.frozenactions.actions.ActionContext;
import org.bukkit.Bukkit;
import org.jetbrains.annotations.NotNull;

public final class ActionbarBroadcastAction implements Action {

    @Override
    public @NotNull String getId() {
        return "ACTIONBARBROADCAST";
    }

    @Override
    public void run(final @NotNull ActionContext context) {
        Bukkit.getOnlinePlayers().forEach(player ->
            player.sendActionBar(context.getData())
        );
    }
}

File: src/main/kotlin/com/github/frcsty/frozenactions/actions/broadcast/ActionbarBroadcastAction.kt
package com.github.frcsty.frozenactions.actions.broadcast

import com.github.frcsty.frozenactions.actions.Action
import com.github.frcsty.frozenactions.util.sendActionBarMessage
import org.bukkit.Bukkit
import org.bukkit.entity.Player

object ActionbarBroadcastAction : Action {
    override val id = "ACTIONBARBROADCAST"

    override fun run(player: Player, data: String) {
        Bukkit.getServer().onlinePlayers.forEach { it.sendActionBarMessage(data) }
    }
}